## parameter for uav_control

control:
  # cascade_pid: P for POS, PID for VEL
  ## 起飞高度
  Takeoff_height : 1.5
  ## 降落速度
  Land_speed : 0.3
  ##　上锁高度
  Disarm_height : 0.1
  ## 最大安全速度
  maximum_safe_vel_xy : 0.2
  maximum_safe_vel_z : 0.5
  ## 最大vision/px4速度误差
  maximum_vel_error_for_vision : 100.0

##　地理围栏
geo_fence:
  x_min: -8.0
  x_max: 8.0
  y_min: -5.0
  y_max: 5.0
  z_min: -0.3
  z_max: 3.0
